src/database/standalone-sqlite.ts

```
@@ -3970,6 +3999,11 @@ if (!isMainThread) {
           const bundle = worker.getBundle(args[0]);
           parentPort?.postMessage(bundle);
           break;
+        case 'isBundleFullyIndexedWithFilters':
+          const bundleFullyIndexed =
+            worker.isBundleFullyIndexedWithFilters(args[0]);
```

Biome (2.4.16)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)